app/products/calls/components/channel_info_start/channel_info_start_button.tsx

```diff
@@ -30,33 +55,59 @@ const ChannelInfoStartButton = ({
     limitRestrictedInfo,
 }: Props) => {
     const intl = useIntl();
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
     const isLimitRestricted = limitRestrictedInfo.limitRestricted;
+    const [connecting, setConnecting] = useState(false);
+    const [joiningMsg, setJoiningMsg] = useState('');
+
+    const starting = intl.formatMessage({id: 'mobile.calls_starting', defaultMessage: 'Starting...'});
+    const joining = intl.formatMessage({id: 'mobile.calls_joining', defaultMessage: 'Joining...'});
```

Comment:
If you recall on webapp we defaulted to `Joining...` to keep it simple (https://github.com/mattermost/mattermost-plugin-calls/pull/684). I am wondering whether we should be consistent on this side.

Reply:
@matthewbirtch what do you think? I'm fine either way, since the code is already written for the more complicated case.

Reply:
I think we should be consistent with the initial button label. If there is no active call we say `Start call`, so `Starting…` makes sense. If there is already an active call the button says `Join call` and so `Joining…` makes sense.

Reply:
I would suggest we keep this and make the web match this behavior

Reply:
Added a ticket to the calls v1 epic: https://mattermost.atlassian.net/browse/MM-58468